main/lsp/LSPPreprocessor.cc

```diff
@@ -49,41 +39,37 @@ string_view LSPPreprocessor::getFileContents(string_view path) const {
     return it->second->source();
 }
 
-void LSPPreprocessor::mergeFileChanges(absl::Mutex &mtx, QueueState &state) {
-    mtx.AssertHeld();
+void LSPPreprocessor::mergeFileChanges() {
```

Comment:
`mergeFileChanges` now operates on a vector of tasks rather than a vector of LSPMessages. The logic is ~identical.